src/components/ProgressGuide/FollowDialog.tsx

```diff
@@ -513,24 +413,24 @@ let Tab = ({
 }): React.ReactNode => {
   const t = useTheme()
   const {_} = useLingui()
-  const activeText = active ? _(msg` (active)`) : ''
+  const label = active
+    ? _(msg`Search for "${interestsDisplayName}" (active)`)
+    : _(msg`Search for "${interestsDisplayName}"`)
```

Comment:
```suggestion
    : _(
        msg({
          message: `Search for "${interestsDisplayName}`,
          comment:
            'Accessibility label for a tab that searches for accounts in a category (e.g. Art, Video Games, Sports, etc.) that are suggested for the user to follow. The tab is not currently active and can be selected.',
        }),
      )
```
nit: And also a suggested comment for translators here